tests/integration_tests/core_tests.py

```diff
@@ -714,10 +714,16 @@ def test_explore_json_async(self):
         keys = list(data.keys())
 
         # If chart is cached, it will return 200, otherwise 202
-        self.assertTrue(rv.status_code in {200, 202})
-        self.assertCountEqual(
-            keys, ["channel_id", "job_id", "user_id", "status", "errors", "result_url"]
-        )
+        assert rv.status_code in {200, 202}
```

Comment:
@villebro I was wondering whether it would be easier/simpler to disable the caching for this test so the response status code is always 200. The only caveat is whether we should be testing both scenarios.